com.unity.multiplayer.mlapi/Runtime/Core/NetworkUpdateManager.cs

```diff
@@ -0,0 +1,320 @@
+                type = currentNUEType
+            };
+            InsertSystem(ref def, networkPostUpdateLoop, typeof(PostLateUpdate.ProcessWebSendMessages));
+
+    #if UNITY_EDITOR
+            PrintPlayerLoop(def);
+    #endif
+
+            PlayerLoop.SetPlayerLoop(def);
+        }
+
+        /// <summary>
+        /// InsertSystem
+        /// Recursively search for the given system type and insert the new system immediately afterwards
+        /// </summary>
+        /// <param name="system">PlayerLoopSystem to search</param>
+        /// <param name="toInsert">PlayerLoopSystem to insert</param>
+        /// <param name="insertAfter">location to insert the PlayerLoopSystem</param>
+        /// <returns></returns>
+        private static bool InsertSystem(ref PlayerLoopSystem system, PlayerLoopSystem toInsert, Type insertAfter)
+        {
+            if (system.subSystemList == null)
+            {
+                return false;
+            }
+
+            for (int i = 0; i < system.subSystemList.Length; ++i)
+            {
+                if (system.subSystemList[i].type == insertAfter)
+                {
+                    InsertSystemAt(ref system, toInsert, i + 1);
+                    return true;
+                }
+            }
+
+            for (var i = 0; i < system.subSystemList.Length; i++)
+            {
+                if (InsertSystem(ref system.subSystemList[i], toInsert, insertAfter))
+                {
+                    return true;
+                }
+            }
+            return false;
+        }
+
+        /// <summary>
+        /// InsertSystemAt
+        /// Copies the subSystemList of the given system and inserts a new system at the given index
+        /// </summary>
+        /// <param name="system">PlayerLoopSystem to be inserted into</param>
+        /// <param name="toInsert">PlayerLoopSystem to insert</param>
+        /// <param name="pos">position to insert the PlayerLoopSystem</param>
+        private static void InsertSystemAt(ref PlayerLoopSystem system, PlayerLoopSystem toInsert, int pos)
+        {
+            PlayerLoopSystem[] newSubSystems = new PlayerLoopSystem[system.subSystemList.Length + 1];
+            for (int i = 0, oldSystemIdx = 0; i < newSubSystems.Length; ++i)
+            {
+                if (i == pos)
+                {
+                    newSubSystems[i] = toInsert;
+                }
+                else
+                {
+                    newSubSystems[i] = system.subSystemList[oldSystemIdx++];
+                }
+            }
+
+            system.subSystemList = newSubSystems;
+        }
+
+        #if UNITY_EDITOR
+        /// <summary>
+        /// PrintPlayerLoop
+        /// Prints all PlayerLoopSystems within the PlayerLoopSystem provided
+        /// </summary>
+        /// <param name="pl">PlayerLoopSystem</param>
+        private static void PrintPlayerLoop(PlayerLoopSystem pl)
```

Comment:
Should share with @SamuelBellomo, might also be very useful if this could appear in the editor